### DISCUSSION TITLE
i am not able to start webpack live server on chrome for frontend changes can you help me how to start the server ?

### DISCUSSION THREAD

**Discussion by roushankumark** [webpack/webpack#18771] [Q&A]
PS C:\Users\roush\Desktop\webpack> npm start

> start
> webpack serve

<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:8080/, http://[::1]:8080/
<i> [webpack-dev-server] On Your Network (IPv4): http://192.168.29.103:8080/
<i> [webpack-dev-server] Content not from webpack is served from 'C:\Users\roush\Desktop\webpack\public' directory
assets by status 130 KiB [cached] 1 asset
orphan modules 41.5 KiB [orphan] 13 modules
runtime modules 27.3 KiB 12 modules
cacheable modules 185 KiB
  modules by path ./node_modules/webpack-dev-server/client/ 81.9 KiB
    ./node_modules/webpack-dev-server/client/index.js?protocol=ws%3A&hostname=0.0.0....(truncated) 52.2 KiB [built] [code generated]
    + 3 modules
  modules by path ./node_modules/webpack/hot/*.js 5.17 KiB
    ./node_modules/webpack/hot/dev-server.js 1.94 KiB [built] [code generated]
    + 3 modules
  modules by path ./node_modules/html-entities/lib/*.js 78.9 KiB
    ./node_modules/html-entities/lib/index.js 4.84 KiB [built] [code generated]
    + 3 modules
  ./src/index.js 393 bytes [built] [code generated]
  ./node_modules/ansi-html-community/index.js 4.16 KiB [built] [code generated]
  ./node_modules/events/events.js 14.5 KiB [built] [code generated]

WARNING in configuration
The 'mode' option has not been set, webpack will fallback to 'production' for this value.
Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/

ERROR in ./src/index.js 1:13-28
Module not found: Error: Can't resolve 'path' in 'C:\Users\roush\Desktop\webpack\src'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
        - install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        - add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
        - install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        - install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "path": false }
resolve 'path' in 'C:\Users\roush\Desktop\webpack\src'
  Parsed request is a module
  using description file: C:\Users\roush\Desktop\webpack\package.json (relative path: ./src)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
    resolve as module
      C:\Users\roush\Desktop\webpack\src\node_modules doesn't exist or is not a directory
      C:\Users\roush\Desktop\webpack\src\node_modules doesn't exist or is not a directory
      looking for modules in C:\Users\roush\Desktop\webpack\node_modules
        single file module
          using description file: C:\Users\roush\Desktop\webpack\package.json (relative path: ./node_modules/path)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\roush\Desktop\webpack\node_modules\path doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\roush\Desktop\webpack\node_modules\path.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\roush\Desktop\webpack\node_modules\path.json doesn't exist
            .wasm
              Field 'browser' doesn't contain a valid alias configuration
              C:\Users\roush\Desktop\webpack\node_modules\path.wasm doesn't exist
        C:\Users\roush\Desktop\webpack\node_modules\path doesn't exist
      C:\Users\roush\Desktop\node_modules doesn't exist or is not a directory
      C:\Users\roush\node_modules doesn't exist or is not a directory
      C:\Users\node_modules doesn't exist or is not a directory
      C:\node_modules doesn't exist or is not a directory

webpack 5.94.0 compiled with 1 error and 1 warning in 762 ms

**Comment by roushankumark** [comment]
I'm passionate about frontend development and eager to get the live server running, but I'm having trouble starting it. Could you guide me through the steps?

**Comment by roushankumark** [comment]
![Uploading Screenshot (141).png…]()

**Comment by alexander-akait** [comment]
You can't use `path` (i.e. `require("path")`/`import ... from "path"`) Node.js module in browsers, we have printed the answer how to solve it, just follow it
